src/FindyIdComponent.js

```diff
@@ -1,24 +1,29 @@
-import React, { Component } from 'react'
+import * as React from 'react'
+import PropTypes from 'prop-types'
 import firebase from 'firebase/app'
 import 'firebase/auth'
 // Load localized firebaseui npm package.
 // See README.md localization section.
 import * as firebaseui from '../lib/npm__ja.js'
 import 'firebaseui/dist/firebaseui.css'
 
-export class FindyIdComponent extends Component {
-  constructor(props) {
-    super(props)
+export class FindyIdComponent extends React.Component {
+  componentDidMount() {
```

Comment:
Start Firebase UI after `didMount` instead of `constructor`.
Target element is not exists on DOM tree on `constructor`.